### PR TITLE
Added release notes for 4.8.0

### PR DESCRIPTION
**Pull request by tmcconechy**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Added notes for 4.8.0 release

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
...
+[Npm Package](https://www.npmjs.com/package/ids-enterprise)\
+[Angular Components Change Log](https://github.com/infor-design/enterprise-ng/blob/master/CHANGELOG.md)
+
+### v4.8.0 Features
+
+- `[Datagrid]` Added an example of Nested Datagrids with ([basic nested grid support.](https://bit.ly/2lGKM4a)) ([#SOHO-3474](https://jira.infor.com/browse/SOHO-3474))
+- `[Datagrid]` Added support for async validation. ([#SOHO-7943](https://jira.infor.com/browse/SOHO-7943))
+- `[Export]` Extracted excel export code so it can be run outside the datagrid. ([#SOHO-7246](https://jira.infor.com/browse/SOHO-7246))
+
+### v4.9.0 Fixes
```
Comment (d-g-h) [inline docs/CHANGELOG.md]: `v4.8.0`?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+
+### v4.8.0 Features
+
+- `[Datagrid]` Added an example of Nested Datagrids with ([basic nested grid support.](https://bit.ly/2lGKM4a)) ([#SOHO-3474](https://jira.infor.com/browse/SOHO-3474))
+- `[Datagrid]` Added support for async validation. ([#SOHO-7943](https://jira.infor.com/browse/SOHO-7943))
+- `[Export]` Extracted excel export code so it can be run outside the datagrid. ([#SOHO-7246](https://jira.infor.com/browse/SOHO-7246))
+
+### v4.8.0 Fixes
+
+- `[Datagrid]` Fixed issues using expand row after hiding/showing columns. ([#SOHO-8103](https://jira.infor.com/browse/SOHO-8103))
```
Comment (EdwardCoyle) [inline docs/CHANGELOG.md]: @tmcconechy This list is missing SOHO-7580 (#161) -- the ToolbarSearchfield/Searchfield API merge

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,74 @@
 # What's New with Enterprise
 
+## v4.8.0
+
+- [Full Release Notes](https://bit.ly/2Ixwfkw)
```
Comment (clepore) [inline docs/CHANGELOG.md]: What does this link to? It just spins for me and never resolves.

File: docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,74 @@
...
+- [Npm Package](https://www.npmjs.com/package/ids-enterprise)
+- [IDS NG (Angular) Change Log](https://github.com/infor-design/enterprise-ng/blob/master/CHANGELOG.md)
```
Comment (clepore) [inline docs/CHANGELOG.md]: Since these changes are in NG, I don't think we need to list this for enterprise.

Comment (tmcconechy) [reply docs/CHANGELOG.md]: We did that at some point because people didnt get that they should look there for angular changes. So i just called it out

**Pull request merged**